docs/tutorials/everything-is-an-event.md

```diff
@@ -0,0 +1,125 @@
+# Everything is an event
+
+This guide will show an example of running the new tracee experience we are experimenting with on Kubernetes.
+For more context on this change, please check the dicussion on github: LINK
+
+## Prerequisites  
+  
+minikube - see installation instructions and dependencies [here](https://minikube.sigs.k8s.io/docs/start/). 
+ 
+*Note: Your local cluster must be run on an intel processor. Apple silicon is not currently supported.* 
+  
+Start minikube and ensure that it is running correctly: 
+ 
+```  
+minikube start 
+ 
+``` 
+ 
+``` 
+kubectl get po -A 
+ 
+NAMESPACE     NAME                               READY   STATUS    RESTARTS   AGE  
+kube-system   coredns-565d847f94-kd9xx           1/1     Running   0          15s  
+kube-system   etcd-minikube                      1/1     Running   0          26s  
+kube-system   kube-apiserver-minikube            1/1     Running   0          26s  
+kube-system   kube-controller-manager-minikube   1/1     Running   0          26s  
+kube-system   kube-proxy-cvqjm                   1/1     Running   0          15s  
+kube-system   kube-scheduler-minikube            1/1     Running   0          26s  
+kube-system   storage-provisioner                1/1     Running   0          15s  
+``` 
+ 
+## Running tracee as a daemon set 
+
+Tracee is installed as a daemonset, and the new experience we are trying is not enabled by default.
+To install we can use helm and pass a custom flag `everythingIsAnEvent`.
+ 
+``` 
+helm install tracee aqua/tracee  --namespace tracee-system --create-namespace  -set everythingIsAnEvent=true 
+
+``` 
+ 
+You can verify that the tracee daemon set is running via: 
+ 
+```  
+kubectl get pods   
+NAME           READY   STATUS    RESTARTS   AGE  
+tracee-fcjmp   1/1     Running   0          4m11s  
+``` 
+  
+## Tracee in action  
+ 
+At this point tracee is running with new behavior. You can see this at the top of the pod’s logs:
+ 
+``` 
+kubectl logs -f daemonset/tracee -n tracee-system 
+``` 
+
+To test it we can trigger a rule, like the anti_debugging one. 
```

Comment:
We keep giving the anti-debugging example, but this example is not a good one. Maybe give another (e.g. fileless)?

Reply:
Yes, but I didn’t have time to prepare the necessary docker image dor a different attack. There is a task for it. Do you have a simple filessimage or instructions we can do here?

Reply:
Is it okay if we use Carolina example? https://github.com/krol3/demo-fileless#demo-using-tracee-and-postee

Reply:
Yes, looks like a good example.